src/qml/PatchManagerPage.qml

```diff
@@ -123,88 +128,17 @@ Page {
         }
     }
 
-    /*! \qmlmethod function showUpdates(manual)
-
-        Flashes the Pulley Menu if updates are available and \a manual is  \c false.
-        Does nothing if \a manual is \c true.
-
-     */
-    function showUpdates(manual) {
-        if (pageStack.busy) {
-            return
-        }
-        if (!manual) {
-            pageStack.busyChanged.disconnect(showUpdates)
-        }
-
-        pulleyAnimation.start()
-    }
-
-    /*! \qmlproperty real PatchManagerPage::pullDownDistance
-        \internal
-        Amount of space the pulley "pops down" when it's showing a hint, e.g the result of showUpdates().
-
-        \warning This is probably broken in recent SFOS versions (?).
```

Comment:
I actually very recently saw this when you updated the Bluetooth patch, and thought: wow, that's nice and wait, that's a bit off -> probably the warning above.

Is this completely removed now? I am not able to see it replaced by anything.

Reply:
Ah, you mentioned it is broken/not anymore present, it is just that the comment wasn't attached to the diff. What can I say, I just _saw_ it working today :))